plenum/server/primary_decider.py

```diff
@@ -76,7 +76,7 @@ def filterMsgs(self, wrappedMsgs: deque) -> deque:
             msg, sender = wrappedMsg
             if hasattr(msg, f.VIEW_NO.nm):
                 reqViewNo = getattr(msg, f.VIEW_NO.nm)
-                if reqViewNo == self.viewNo:
+                if reqViewNo >= self.viewNo:
```

Comment:
`self.viewNo` is set by `startViewChange`, so once that is done, this method will filter correctly. Even if a node is say in view 10 and starts getting enough view 20 messages, ``startViewChange` will be triggered by `_start_view_change_if_possible` which in turn is triggered by `msgHasAcceptableViewNo`